src/components/FactCard.tsx

```diff
@@ -0,0 +1,20 @@
+import { Section } from "../data/factData";
+import "./FactCard.scss";
+
+const FactCard = (props: { section: Section }) => {
+  return (
+    <div className="fact-section">
+      {props.section.facts.map((fact) => (
+        <div className="fact-card">
+          <h3>{fact.factTitle}</h3>
+          <div className="card-content">
+            <img className="fact-image" src={fact.factImage} alt="factImage" />
```

Comment:
The alt-text needs to be descriptive - I know this is a pain, but somewhere you should have a description of all the images and pass it in here (this is really boring and we don't have much time, so I'm happy for you to skip it this time, but you do need to know what the right thing to do is)

Reply:
understood for future.